examples/verifying_mfd.py

```diff
@@ -1,16 +1,177 @@
+          "(with respect to a fixed FD) is a collection of rows that share "
+          "the same left-hand side, but differ in the right-hand side.")
+    print("Let's take a closer look at the indicated clusters.\n")
+    print('"Don Quixote" has been found with 3 different durations: 135, 139 '
+          "and 140. From this, we can conclude that "
+          "(assuming euclidian distance), the maximum distance between "
+          "values is 140 - 135 = 5, which exceeds the provided "
+          "parameter of δ = 3. When we tried to verify the same MFD on this "
+          "dataset with δ = 5, this cluster adhered to the MFD as 5 <= δ, "
+          "and the whole MFD held.")
+    print('"Romeo and Juliet" had the exact same issue: the biggest '
+          "difference between durations of different versions appeared to "
+          "have been 165 - 160 = 5, which held with greater parameter value.")
+    print("-"*80)
+
+def mfd_strings():
+    METRIC = "cosine"
+    METRIC_ALGORITHM = "brute"
+    LHS_INDICES = [1]
+    RHS_INDICES = [2]
+    PARAMETERS = [0.75, 0.5, 0.25, 0.1, 0.01]
+    TABLE="examples/datasets/addresses_names.csv"
+    bigrams_table="examples/datasets/bigrams.csv"
+
+    print("Let's showcase this by checking addresses_names.csv and "
+          "trying to verify the metric functional dependency in it.\n")
+
+    check_mfd(LHS_INDICES, RHS_INDICES, METRIC, PARAMETERS, TABLE,
+              "[SSN] -> [Address]", METRIC_ALGORITHM)
+
+    print("To get into the intricacies of how the distance has been "
+          "calculated here, we need to first define what is known as an "
+          f"n-gram. {BLUE_CODE}N-grams{DEFAULT_COLOR_CODE} are collections "
+          "of adjacent symbols of fixed "
+          "size, such as 'aab' or 'ba'. Depending on the length, they are "
+          "called bigrams (2), trigrams (3) and so on.\n")
+    print("To turn this definition into a useful tool, let's make a table "
+          "containing a row for every row in addresses_names.csv, and a "
+          "column for every bigram occuring in that table. The number of "
+          "occurences will be shown as a value in (row;bigram). "
+          "Our example would contain two rows and 30 columns:")
+
+    bigrams = pd.read_csv(bigrams_table, header=[0])
+    print(bigrams)
+
+    print("If you were to plot two vectors in a 30-dimensional space, and "
+          "then calculate an angle between those two vectors, the cosine of "
+          "that angle would be a cosine distance on bigrams.")
+    print("This algorithm builds on top of that simple principle, but the "
+          "gist of it is that the difference in occurences of bigrams leads "
+          "to those strings being perceived as 'different', or, in metric "
+          "terms, 'distant', providing us with a way of measuring how much "
+          "a particular string deviates from some other, similar string.\n")
```

Comment:
(unless you're sure that "differs" is the right term, I may be lacking some info here)
```suggestion
          "a particular string differs from another, similar string.\n")
```

Reply:
"providing us with a way of measuring the difference between two strings."